web/src/main/java/com/futurewei/alcor/web/restclient/ElasticIpManagerRestClient.java

```diff
@@ -1,11 +1,77 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+
 package com.futurewei.alcor.web.restclient;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.futurewei.alcor.common.stats.DurationStatistics;
+import com.futurewei.alcor.common.utils.ControllerUtil;
+import com.futurewei.alcor.web.entity.elasticip.ElasticIpInfo;
+import com.futurewei.alcor.web.entity.elasticip.ElasticIpInfoWrapper;
+import com.futurewei.alcor.web.entity.elasticip.ElasticIpsInfoWrapper;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.*;
+import org.springframework.util.MimeType;
+import org.springframework.util.MimeTypeUtils;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Map;
 
 public class ElasticIpManagerRestClient extends AbstractRestClient {

```

Comment:
Oh EIP calls itself! Could we handle those calls internally or ask other instance to handle is better?

Reply:
Seems I don't get the point... This rest client is for port microservice, when delete a port or a fixed ip of it is removed, the port microservice should notify elastic ip manager to remove the related associations.

Reply:
hmm, I thought this client is for EIP mgr to call itself. Never mind then.